SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -139,6 +143,15 @@ void EntityFactory::WriteLBComponents(TArray<FWorkerComponentData>& ComponentDat
 	ComponentDatas.Add(NetOwningClientWorker(AuthoritativeClientPartitionId).CreateComponentData());
 	ComponentDatas.Add(AuthorityIntent(IntendedVirtualWorkerId).CreateComponentData());
 	ComponentDatas.Add(AuthorityDelegation(DelegationMap).CreateComponentData());
+
+	if (GetDefault<USpatialGDKSettings>()->bEnableStrategyLoadBalancingComponents)
+	{
+		const ActorSetWriter ActorSetWriter(NetDriver->Connection->GetCoordinator(), *NetDriver->PackageMap);
```

Comment:
Don't use the same names for fields as for the types.

Reply:
A little sad we let `F` prefix go - `FActorSetWriter ActorSetWriter` feels more natural than `ActorSetWriter SetWriter`.